2021-offseason/src/main/java/frc/robot/RobotContainer.java

```diff
@@ -22,6 +25,7 @@
 public class RobotContainer {
   // The robot's subsystems and commands are defined here...
   private final ExampleSubsystem m_exampleSubsystem = new ExampleSubsystem();
+  private final Flywheel m_flywheel = new Flywheel();
```

Comment:
member variable declared but not used

Reply:
its for the future when I finish out this stuff after writing commands